src/core/thread.d

```diff
@@ -1860,6 +1868,8 @@ private:
             return;
```

Comment:
Seems that this was used previously to see if the thread was detached. You can either replace this
with `if (t.m_detached == false)` or use this in the destructor directly.